pyvista/plotting/_vtk.py

```diff
@@ -0,0 +1,108 @@
+"""
+Import from vtk.
+
+These are the modules within vtk that must be loaded across pyvista.
+Here, we attempt to import modules using the ``vtkmodules``
+package, which lets us only have to import from select modules and not
+the entire library.
```

Comment:
Good point, will do

> Actually, I'd go so far as to rename _vtk.py to _vtk_plotting.py. If anything, I'd expect the unqualified _vtk to be the core one...

I generally agree but am reluctant to fo this as this time. This is because of the `*` imports and generally trying to keep the namespace the same as it previously was. This way `pyvista._vtk` is still all VTK imports, GL-dependent and all.

When I eventually split the core and plotting modules into separate packages, both these modules will just be called `_vtk.py` in their respective packages.